### DISCUSSION TITLE
Setting up v1 with composition Api

### DISCUSSION THREAD

**Discussion by Asgen** [quasarframework/quasar#10526] [Quasar App Extensions]
//quasar.conf.js
    ...
    supportTS: true
    ...

//tsconfig.json
     {
       "extends": "@quasar/app/tsconfig-preset",
       "compilerOptions": {
         "baseUrl": "."
       }
     }

With this settings I've got errors `Property or method "navigationLinks" is not defined on the instance but referenced during render. Make sure that this property is reactive` in components:

//src/layouts/MainLayout.vue
<img width="678" alt="Screen Shot 2021-08-30 at 21 43 37" src="https://user-images.githubusercontent.com/30123749/131388750-ced3302a-4d35-4a80-914f-2118f170e0a7.png">

What's wrong?

**Comment by ontwikkelfabriek** [comment]
@Asgen 

Could you create a codepen.io so we can take a look at it and give better support?
